Source/GameView/Repositories/GameAPIRepo.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  GameAPIRepo.swift
+//  CommonFiles
+//
+//  Created by Tiewhan Smith on 2020/03/06.
+//
+
+import Foundation
+
+///Structure is only used to model JSON respones. Contains the a structure of Apps
+struct AppList: Codable {
+  let applist: Apps
+}
+
+///Structure is used to model JSON respones. Contains the list of games
+struct Apps: Codable {
+  let apps: [JsonGame]
+}
+
+///Structure is used to model JSON respones. This is the actual game
+struct JsonGame: Codable {
+  let appid: Int
+  let name: String
+}
+
+public class GameAPIRepo: GameRepository {
+  
+  private var observers: [String: GameRepositoryObserver] = [:]
+  
+  public init() {
+    
+  }
+  
+  public func getGameList(with serviceCaller: ServiceCaller) {
+
+    let url: URL = URL(string: SteamURLComponents.gameListURL)!
+    
+    // MARK: - Call Failed Closure
+    serviceCaller.callFailed = { error in
+      
```

Comment:
Why don't we do anything with this error ?

Reply:
Simply Put, I had more pressing issues at the time of writing.

Reply:
Doesn't have to be resolved immediately but, usually, the ticket needs to be fully completed before opening a PR.  I'm aware that's not the case here, just pointing it out for the future